templates/agent-template-aro-hcp.yml

```diff
@@ -33,6 +38,11 @@ parameters:
   displayName: Image tag
   value: latest
 
+- name: KLOG_V
+  displayName: KLOG V Level
+  description: Log verbosity level
+  value: "1"
```

Comment:
may be 3 by default?

Reply:
- updated default `KLOG_V` environment in template to `3`.
- added comments to explain the mapping between klog level and zap log level.